client/src/pages/admin/auto-test-task/[taskId].tsx

```diff
@@ -0,0 +1,149 @@
+import { Checkbox, Descriptions, Divider, List, Space, Switch, Tag, Typography } from 'antd';
+import { AutoTestTaskDto, AutoTestsApi } from 'api';
+import { AdminPageLayout } from 'components/PageLayout';
+import { ActiveCourseProvider, SessionProvider, useActiveCourseContext } from 'modules/Course/contexts';
+import { CourseRole } from 'services/models';
+import { GetServerSideProps } from 'next';
+import { getTokenFromContext } from 'utils/server';
+import { getApiConfiguration } from 'utils/axios';
+
+export interface AutoTestTask extends AutoTestTaskDto {
+  attributes: Record<string, any>;
+}
+
+type PageProps = {
+  selectedTask: AutoTestTask;
+};
+
+export const getServerSideProps: GetServerSideProps = async context => {
+  try {
+    context.params;
```

Comment:
prosze to usunąć

Reply:
Dziękuję! Usunąłem ten kod. Z nadchodzącym Nowym Rokiem!